---
title: (24-01-25) Successful Master research project defense (MSc. Data Science for Decision Making and MSc. Artificial Intelligence)
summary: Ludwig Schulte, Mikolaj Czaplejewicz, Dhruv sharma and Jonathon Bird successfully defended master thesis project on "GWAS IE - Information extraction for genome wide association studies". The group developed an LLM-based automated information extraction system to identify key biological entities and their relationships from scientific literature and translate this information in the form of a knowledge graph. The resulting tool aids researchers in rapidly uncovering and organizing genotype–phenotype insights from GWAS papers, demonstrating both high accuracy and strong coverage in a specialized scientific domain. This work was jointly supervised by Prof. Wim Vriezen (Head, Plant Functional Genomics BFFI Maastricht University), Dr. Aki Harma (Assistant Professor DACS, Maastricht University) and myself. Congratulations to the team on their successful defense. Good luck for future scientific adventures.   

tags:
- transcriptomics
- solanum lycopersicum
- plants
- defense responses 
date: "2025-01-25T00:00:00Z"

# Optional external URL for project (replaces project detail page).
external_link: ""

image:
  caption: Project report front page
  focal_point: Smart

links:
- icon: twitter
  icon_pack: fab
  name: Follow
  url: https://twitter.com/kumar_s_singh
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

posters: ""
# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""
---
## Abstract
This work presents a system for automated information extraction from Genome-Wide As- sociation Studies (GWAS) literature, focusing on gene variants and their associated traits. By leveraging a Unified Information Extraction (UIE) model and a domain-tailored schema, our approach systematically identifies key entities (e.g., gene variants and phenotypes) and their relationships, storing them in a searchable knowledge graph. The resulting tool aids researchers in rapidly uncovering and organizing genotype–phenotype insights from GWAS papers, demon- strating both high accuracy and strong coverage in a specialized scientific domain.
