---
title: "Evolution of Mutator transposable elements across eukaryotic diversity"

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here 
# and it will be replaced with their full name and linked to their profile.
authors:
- admin

# Author notes (optional)
author_notes:
- " "

date: "2019-03-21T00:00:00Z"
doi: "10.1186/s13100-019-0153-8"

# Schedule page publish date (NOT publication's date).
publishDate: "2019-03-21T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: Mobile DNA
publication_short:

abstract: Mutator-like elements (MULEs) are a significant superfamily of DNA transposons on account of their (i) great transpositional activity and propensity for insertion in or near gene sequences, (ii) their consequent high mutagenic capacity, and, (iii) their tendency to acquire host gene fragments. Consequently, MULEs are important genetic tools and represent a key study system for research into host-transposon interactions. Yet, while several studies have focused on the impacts of MULEs on crop and fungus genomes, their evolution remains poorly explored. We perform comprehensive bioinformatic and phylogenetic analyses to address currently available MULE diversity and reconstruct evolution for the group. For this, we mine MULEs from online databases, and combine search results with available transposase sequences retrieved from previously published studies. Our analyses uncover two entirely new MULE clades that contain elements almost entirely restricted to arthropod hosts, considerably expanding the set of MULEs known from this group, suggesting that many additional MULEs may await discovery from further arthropod genomes. In several cases, close relationships occur between MULEs recovered from distantly related host organisms, suggesting that horizontal transfer events may have played an important role in the evolution of the group. However, it is apparent that MULEs from plants remain separate from MULEs identified from other host groups. MULE structure varies considerably across phylogeny, and TIR length is shown to vary greatly both within and between MULE groups. Our phylogeny suggests that MULE diversity is clustered in well-supported groups, typically according to host taxonomy. With reference to this, we make suggestions on how MULE diversity can be partitioned to provide a robust taxonomic framework. Our study represents a considerable advance in the understanding of MULE diversity, host range and evolution, and provides a taxonomic framework for the classification of further MULE elements that await discovery. Our findings also raise a number of questions relating to MULE biology, suggesting that this group will provide a rich avenue for future study.

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags: []

# Display this page in the Featured widget?
featured: false

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

url_pdf: 'https://mobilednajournal.biomedcentral.com/track/pdf/10.1186/s13100-019-0153-8.pdf'
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

---

{{% callout note %}}
Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /callout %}}

{{% callout note %}}
Create your slides in Markdown - click the *Slides* button to check out the example.
{{% /callout %}}
