---
title: (25-04-24) Successful master thesis defense by Erika Castaneda Sastre
summary: Erika Castaneda Sastre defended her master dissertation (Master of Science) on the topic "WYtigger - Identifying autosomal and sex-linked contigs using a machine learning-based classifier". This work was jointly supervised by Prof. Dick de Ridder (Bioinformatics, WUR) and myself. Congratulations to Erika on her successful defense and good luck for future scientific adventures.   

tags:
- machine learning
- sex chromosomes
- plants
- classifier 
date: "2024-04-25T00:00:00Z"

# Optional external URL for project (replaces project detail page).
external_link: ""

image:
  caption: Master thesis front page
  focal_point: Smart

links:
- icon: twitter
  icon_pack: fab
  name: Follow
  url: https://twitter.com/kumar_s_singh
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

posters: ""
# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""
---
## Abstract
Chromosomes, carriers of an organism's hereditary information, can be categorized into two main types: allosomes and autosomes. Allosomes, also known as sex chromosomes, play a crucial role in sex determination and regulating sex-related traits. Despite considerable diversity, they share standard features and exhibit variations in gene content and pairing systems. Understanding sex chromosomes is crucial for agricultural and disease control efforts, where genetic approaches utilizing sex-specific traits show promise. However, identifying allosomes, especially in non-model organisms, poses challenges. Here, we explore the use of supervised machine learning models, including logistic regression, random forests, support vector machines, and k-nearest neighbors, to classify contigs as autosomes or allosomes based on whole-genome sequencing data. The predictive capability of features like coverage, heterozygosity, and GC content is assessed. The results underscore the importance of feature combinations and model selection for accurate classification.

 


